easybuild/framework/extension.py

```diff
@@ -172,24 +172,64 @@ def version(self):
         return self.ext.get('version', None)
 
     def prerun(self):
+        """
+        [DEPRECATED] Stuff to do before installing a extension.
+        """
+        self.log.deprecated(
+            "Extension.prerun() is deprecated, use Extension.pre_install_extension() instead.",
+            '5.0',
```

Comment:
@lexming This should be `6.0`, otherwise hitting `prerun` will raise an error, rather than only print a deprecation warning

Reply:
fixed in 21f1d58